### PR TITLE
chore(travis): add Node 14 to Travis and update amex-jest-preset

### PR DESCRIPTION
**Pull request by JamesSingleton**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
     - stage: release
-      node_js:
-        - 10
-        - 12
+      node_js: lts/*
       deploy:
         on:
           branch: master
         provider: script
-        skip_cleanup: true
```
Comment (JAdshead) [inline .travis.yml:17]: should this be `cleanup: false` now ?

Comment (JamesSingleton) [reply .travis.yml:17]: Nope, https://blog.travis-ci.com/2019-08-27-deployment-tooling-dpl-v2-preview-release

File: __tests__/index.spec.js [hunk 2]
```diff
@@ -517,7 +518,23 @@ describe('one-app-dev-cdn', () => {
...
-      expect(got.mock.calls).toMatchSnapshot('module response');
+      expect(got.mock.calls).toEqual([
+        [
+          'https://example.com/module-map.json',
+          {
+            agent: expect.any(ProxyAgent),
+          },
+        ],
+        [
+          'https://example.com//cdn/module-b/1.0.0/module-b.node.js',
```
Comment (JAdshead) [inline __tests__/index.spec.js:529]: where is the `//` coming from ?
```suggestion
          'https://example.com/cdn/module-b/1.0.0/module-b.node.js',
```

Comment (JAdshead) [reply __tests__/index.spec.js:529]: this should be looked at and fixed in a followup pr

**Pull request merged**